app/src/main/kotlin/nl/eduid/screens/homepage/HomePageWithAccountContent.kt

```diff
@@ -67,6 +77,26 @@ fun HomePageWithAccountContent(
     val sheetState = rememberModalBottomSheetState()
     var showBottomSheet by remember { mutableStateOf(false) }
 
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
```

Comment:
> I think we should handle this also on the HomePageWithoutAccount
That would result in a permission request immediately on first launch.
Also that homescreen is shown if you have no identity. This one is shown if you have at least one, which is the only case when you can get pushes.

>  If you use the accompanist permission you can wrap it with
I opted out of using it, because I did not want to import a new library just for these few lines of code.

> With if (LocalInspectionMode.current) return you also don't break the preview.
Previews are working fine, no need for that